models/contests.js

```diff
@@ -1,3 +1,4 @@
+import "./contests.css".
```

Comment:
You should not import any css in the models
This is just the mongodb model.

Also I think there no any contests.css so remove this from here.